ballerina/target/cache/tests_cache/test_suit.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Shall we remove this file

Reply:
Fixed in 3d6703186427b015a571e913cdbbf57c88c7816d